README.rst

```diff
@@ -375,6 +375,55 @@ Usage exmaple:
     >>> user.organizations_dict.keys()
     ... dict_keys(['20135c30-d486-4d68-993f-322b8acb51c4'])
 
+Organization membership permission_classes
+------------------------------------------
+
+The custom rest_framework permission classes ``IsOrganizationMember``,
+``IsOrganizationManager`` and ``IsOrganizationOwner`` can be used in the
+API to ensure that the request user is in the same organization as requested
+object and is organization member, manager or owner respectively.
+
+Usage exmaple:
+
+.. code-block:: python
+
+    from openwisp_users.multitenancy import IsOrganizationManager
+
+    class MyApiView:
+        permission_classes = (IsOrganizationMember,)
+
+        def get(self, request, *args, **kwargs):
+            # Perform all the actions required and
+            # get the requested object in variable ``obj``
+            # and call check_object_permissions which returns
+            # ``403`` if the user is not allowed to access
+            # the requested object.
+            self.check_object_permissions(request, obj)
+
+organization_field
+~~~~~~~~~~~~~~~~~~
+
++--------------+------------------+
+| **type**:    | ``string``       |
++--------------+------------------+
+| **default**: | ``organization`` |
++--------------+------------------+
+
+When using the Organization Member permission_classes, you can use
+``organization_field`` to describe the relationship between object
+and organization:
+
+.. code-block:: python
+
+    from openwisp_users.multitenancy import IsOrganizationManager
+
+    class MyApiView:
+        permission_classes = (IsOrganizationMember,)
+        organization_field = 'object_parent__organization'
+
+This will get organization from object as ``obj.object_parent__organization``
+and check if the users is has membership in that organization.
+
```

Comment:
Try to go more straight to the point and to provide a real world example: 

```
``organization_field`` can be used to define where to look to find the organization of the current object.

In most cases this won't need to be changed, but it does need to be changed when the ``organization`` is 
defined only on a parent object.

For example, in openwisp-firmware-upgrader (link), ``organization`` is defined on ``Category`` and ``Build`` has a relation to ``category``, so the organization of Build instances is inferred from the organization of the Category.

Therefore, to implement the permission class correctly, we would have to do:

......
organization_field = 'category__organization'

This will translate into accessing ``obj.category.organization``.
Ensure the queryset of your views make use of select_related (link django docs) in these cases to avoid generating too many queries.
```